engines/bladerunner/bladerunner.cpp

```diff
@@ -360,7 +360,7 @@ Common::Error BladeRunnerEngine::run() {
 
 	if (!startup(hasSavegames)) {
 		// shutting down
-		return Common::Error(Common::kUnknownError, _("Failed to initialize resources"));
+		return Common::Error(Common::kUnknownError, _s("Failed to initialize resources"));
```

Comment:
@criezy It seems that `Common::Error` is used localized.
I am not used to this class so some check may be needed there.